src/SFML/Window/OSX/SFOpenGLView.mm

```diff
@@ -27,29 +27,14 @@
 // Headers
 ////////////////////////////////////////////////////////////
 #include <SFML/Window/OSX/WindowImplCocoa.hpp>
-#include <SFML/Window/OSX/HIDInputManager.hpp> // For localizedKeys and nonLocalizedKeys
 #include <SFML/System/Err.hpp>
 
-#import <SFML/Window/OSX/SFKeyboardModifiersHelper.h>
 #import <SFML/Window/OSX/SFOpenGLView.h>
+#import <SFML/Window/OSX/SFOpenGLView+mouse_priv.h>
```

Comment:
Is that a common convention for Objective C (I have no idea...)? Otherwise, I'd probably stick to SFML's existing one, i.e. SFOpenGLViewMousePriv or something. But if the `+` makes a crucial part of the semantics, we can also just leave it.